.github/workflows/ci.yml

```diff
@@ -0,0 +1,257 @@
+---
+name: Continuous Integration tests
+
+on: [pull_request]
+
+jobs:
+  main_tests:
+    name: Main tests
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        python-version: ['3.9']
+        # ['2.7', '3.6', '3.7', '3.8', '3.9', '3.10.0-rc.1', 'pypy-2.7', 'pypy-3.6', 'pypy-3.7-nightly']
+        include:
```

Comment:
It might be better to just skip the matrix above and manually declare the permutations as done below?

Reply:
I don't have any preference. In theory, if we declare that we support a Python version the tests should pass on all the supported OSes so the matrix looks better.
Do you have any specific reason to prefer the explicit list?

Reply:
Not especially.  The only reason would be that if the matrix is only going to be 1x3 then it doesn't serve very much purpose - but that's not a real *problem* per se.

Reply:
no, I think we want to support multiple Python versions so it will be 3xN at least.
+1 for the matrix